crates/sql/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "slight-sql"
+version = "0.1.0"
+edition = { workspace = true }
+authors = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
+wit-bindgen-wasmtime = { workspace = true }
+wit-error-rs = { workspace = true }
+slight-common = { path = "../common" }
+slight-runtime-configs = { path = "../runtime-configs" }
+anyhow = { workspace = true }
+tracing = { workspace = true }
+tokio = { workspace = true }
+async-trait = { workspace = true }
+# sql.postgres deps
+postgres = { version = "0.19", optional = true }
```

Comment:
From this description https://docs.rs/postgres/latest/postgres/index.html#implementation, it looks like to me `tokio_postgres` may be a better library to use since we are using tokio runtime?

Reply:
`postgres` is simply a wrapper, we should be ok going off of that.

Reply:
"The client simply blocks on the futures provided by the async client." Since we are running postgres API in async function, we actually don't want the library to block on the future, no?